stable/kong/templates/service-kong-admission-webhook.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.ingressController.admissionWebhook.enabled }}
+apiVersion: v1
```

Comment:
Can we keep a single file for all things admission webhook so that there is a single `if` guarding it?
Again, a historical reason for code management.
(Who knew maintaining a YAML templating code-base would get this complicated?)

Reply:
Done. I originally put it in its own service-*.yaml file to match the other services, but putting it all in one webhook yaml file makes more sense.